prune/prune_test.go

```diff
@@ -210,7 +210,7 @@ var _ = Describe("Prune", func() {
 					Expect(len(pods.Items)).Should(Equal(1))
 				})
 
-				It("Should Prune Jobs with Default IsPrunableFunc", func() {
+				It("Should Prune Jobs with Default IsPrunableFunc", func() { //nolint:dupl
```

Comment:
why the code is duplicated?
Could we not solve it instead of ignoring the lint check?
Also, why do we have more than one test with the same description name?

Reply:
There shouldn't be more than one test with the same description name. One should be "Should Prune Pods with Default IsPrunableFunc" and one should be "Should Prune Jobs with DefaultIsPrunableFunc".

I will create a function that both the tests can use since they pretty much do the exact same thing but use slightly different values.

Reply:
Should be resolved in https://github.com/operator-framework/operator-lib/pull/105/commits/264c78d7aa3a87b4191cff6e09f0a7d1b94cb08e